tempus_fugit_minecraft/player.py

```diff
@@ -133,6 +137,15 @@ def speed_down(self) -> None:
         """
         if self.walking_speed > 5:
             self.walking_speed -= self.WALK_SPEED_INCREMENT
+   
+    #Issue #39
+    def increase_jump(self) -> None:       
+            if self.jump_speed <= self.MAX_JUMP_SPEED: 
+                self.jump_speed += 5
+    #Issue #39
+    def decrease_jump(self) -> None:
+        if self.jump_speed >= self.MIN_JUMP_SPEED:       
+            self.jump_speed -= 5
```

Comment:
Since we're modifying the jump speed, maybe we should call these `increase_jump_speed` and `decrease_jump_speed`.

Reply:
Also, please add doxygen style comments similar to @bakursait2's PR

Reply:
> Also, please add doxygen style comments similar to @bakursait2's PR

Done!